lldb/tools/lldb-dap/Handler/RequestHandler.cpp

```diff
@@ -57,9 +57,8 @@ void BaseRequestHandler::SetSourceMapFromArguments(
       "source must be be an array of two-element arrays, "
```

Comment:
Just FYI, the `launch` command has been updated and is going through https://github.com/llvm/llvm-project/blob/d913ea307e32aa1412b67d02f2f76a5f2083f2e8/lldb/tools/lldb-dap/DAP.cpp#L1194 for setting the target.source-map value. I've been working on migrating the requests to not use raw json objects but I haven't updated the attach handler yet.

The test, as currently written, is not actually going to run this code. Its still good to have and check though.